package.json

```diff
@@ -7,7 +7,7 @@
     "lib": "lib"
   },
   "scripts": {
```

Comment:
If you change your test script to "jest --verbose --coverage" it will run all of your tests